build.sbt

```diff
@@ -39,7 +39,7 @@ def buildLevelSettings: Seq[Setting[_]] =
       homepage := Some(url("https://github.com/sbt/sbt")),
       scmInfo := Some(ScmInfo(url("https://github.com/sbt/sbt"), "git@github.com:sbt/sbt.git")),
       resolvers += Resolver.mavenLocal,
-      scalafmtOnCompile := true,
+      scalafmtOnCompile := false,
```

Comment:
Could you update CONTRIBUTING and maybe even the `onLoadMessage` to tell ppl to run mima check and Scalafmt commands before committing?